src/pages/gen2/reference/cli-commands/index.mdx

```diff
@@ -224,3 +224,17 @@ Generate GraphQL statements for your frontend application to consume.
 Generate React form components derived from your backend data models for your frontend application to consume.
 
 {/* @TODO */}
+
+## npx amplify info
+
+Generates information on system, binaries, NPM packages and environment variables for troubleshooting Amplify issues.
+
+## npx amplify pipeline-deploy
```

Comment:
can we add example usage for this one? we'll likely come back and link out to something that goes into more detail, but a quick one-liner would suffice, or the default that's provided by Hosting's `amplify.yml`